usagov/vote-gov-drupal
# VOTE-2503 content editor role permission adjustments

VOTE-1987 site builder role permission simplification



## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-2503
https://cm-jira.usa.gov/browse/VOTE-1987

## Description

We are update permissions for the roles: Content Editor and Site Builder.
The update for the site builder role is in preparation of VOTE-1987

## Deployment and testing

### Post-deploy steps

1. `lando retune`

### QA/Testing instructions

1. create a new user with role of Site Builder and test to make sure they have access to the same as the Site Admin.
2. create a new user with role of Content Editor and test to make sure they can only edit Basic Pages, Voter Guides and taxonomy term translations. This one is a little complicated so I'm happy to provide more information if needed.

## Checklist for the Developer




## Checklist for the Peer Reviewers
